backend/functions/lib/adapters/gateways/dxtr/orders.dxtr.gateway.ts

```diff
@@ -58,42 +92,21 @@ export class DxtrOrdersGateway implements OrdersGateway {
           return rawOrder satisfies Order;
         })
         .sort(dxtrOrdersSorter);
+      context.logger.info(
```

Comment:
This is getting flag by Veracode. Using the log service should resolve this
`import log from '../lib/adapters/services/logger.service';`